src/main/java/edu/nps/moves/dis7/ElectronicEmissionsPdu.java

```diff
@@ -1,327 +1,365 @@
-   protected short  stateUpdateIndicator;
+    /**
+     * This field shall be used to indicate if the data in the PDU represents a state update or just data that has changed since issuance of
+     * the last Electromagnetic Emission PDU [relative to the identified entity and emission system(s)].
+     */
+    protected short stateUpdateIndicator;
 
-   /** This field shall specify the number of emission systems being described in the current PDU. */
-   protected short  numberOfSystems;
+    /** This field shall specify the number of emission systems being described in the current PDU. */
+    protected short numberOfSystems;
 
-   /** padding */
-   protected int  paddingForEmissionsPdu;
+    /** padding */
+    protected int paddingForEmissionsPdu;
 
-   /**  this field shall specify the length of this emitter system's data in 32-bit words. */
-   protected short  systemDataLength;
+    /** this field shall specify the length of this emitter system's data in 32-bit words. */
+    protected short systemDataLength;
 
-   /** the number of beams being described in the current PDU for the emitter system being described.  */
-   protected short  numberOfBeams;
+    /** the number of beams being described in the current PDU for the emitter system being described. */
+    protected short numberOfBeams;
 
-   /**  information about a particular emitter system and shall be represented by an Emitter System record (see 6.2.23). */
-   protected EmitterSystem  emitterSystem = new EmitterSystem(); 
+    /** information about a particular emitter system and shall be represented by an Emitter System record (see 6.2.23). */
+    protected EmitterSystem emitterSystem = new EmitterSystem();
 
-   /** the location of the antenna beam source with respect to the emitting entity's coordinate system. This location shall be the origin of the emitter coordinate system that shall have the same orientation as the entity coordinate system. This field shall be represented by an Entity Coordinate Vector record see 6.2.95  */
-   protected Vector3Float  location = new Vector3Float(); 
+    /**
+     * the location of the antenna beam source with respect to the emitting entity's coordinate system. This location shall be the origin of
+     * the emitter coordinate system that shall have the same orientation as the entity coordinate system. This field shall be represented
+     * by an Entity Coordinate Vector record see 6.2.95
+     */
+    protected Vector3Float location = new Vector3Float();
 
-   /** Electronic emmissions systems THIS IS WRONG. It has the WRONG class type and will cause problems in any marshalling. */
-   protected List< Vector3Float > systems = new ArrayList< Vector3Float >(); 
+    /** Electronic emmissions systems THIS IS WRONG. It has the WRONG class type and will cause problems in any marshalling. */
+    protected List<Vector3Float> systems = new ArrayList<>();
```

Comment:
Is the above comment still accurate or is it fixed now with your change?

Reply:
Fixed in commit https://github.com/open-dis/open-dis-java/pull/76/commits/5cbbbbdb9c2d243a90fb21344863b1812259d513